catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/operation/impl/UpdateRequestImpl.java

```diff
@@ -92,6 +92,23 @@ public UpdateRequestImpl(String[] ids, List<Metacard> metacards)
     }
 
     /**
+     * Instantiates a new UpdateRequestImpl from a set of destinations,
+     * an array of ids, and a list of {@link Metacard}
+     *
+     * @param destinations the destination ids this request should be sent to
+     * @param ids          the ids associated with the {@link Metacard} list
```

Comment:
These are the metacard IDs, but this constructor was built this way so we can utilize some of the other functionality in `UpdateRequestImpl` such as `formatEntryList`.  Since this class stores the Metacards as a List<Entry>, this seemed to be the best approach.